tests/test_templates_layers.py

```diff
@@ -462,3 +462,75 @@ def test_random_layer_weights(self, n_subsystems, tol):
         params = [q.parameters for q in rec.queue]
         params_flat = [item for p in params for item in p]
         assert np.allclose(weights.flatten(), params_flat, atol=tol)
+
+
+class TestCnotRing:
+    """Tests for the CnotRingLayers method from the pennylane.templates.layers module."""
+
+    @pytest.mark.parametrize("n_wires, n_cnots", [(1, 0),
+                                                  (2, 1),
+                                                  (3, 3),
+                                                  (4, 4)])
+    def test_circuit_queue(self, n_wires, n_cnots):
+        """Tests the gate types and parameter values in the circuit."""
+        np.random.seed(12)
```

Comment:
How come seeding is used here?

Reply:
I think that's a copy and paste error, thanks!